products/istio.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Istio
+category: server-app
+iconSlug: istio
+permalink: /istio
+versionCommand: istioctl version
+releasePolicyLink: https://istio.io/latest/docs/releases/supported-releases/#support-policy
+changelogTemplate: https://istio.io/latest/news/releases/__RELEASE_CYCLE__.x/announcing-__LATEST__/
+activeSupportColumn: false
+releaseDateColumn: true
+
+auto:
+-   git: https://github.com/istio/istio.git
```

Comment:
The distroless one doesn't seem to follow istio-versioning (No 1.17.1 tag for eg) - so it might cause issues with scanners.

I've picked the images that I think are used in production deployments. Please correct if I'm wrong about any of these.

```suggestion
-   git: https://github.com/istio/istio.git
-   purl: pkg:docker/istio/pilot
-   purl: pkg:docker/istio/distroless
-   purl: pkg:docker/istio/install-cni
-   purl: pkg:docker/istio/app
-   purl: pkg:docker/istio/proxyv2
-   purl: pkg:docker/istio/operator
-   repology: istio
-   repology: istioctl
-   repology: go:istio-pkg
-   repology: go:istio-api
```